apps/code-examples/src/app/code-examples/layout/box/inline-help/box-demo.component.html

```diff
@@ -0,0 +1,23 @@
+<sky-box>
+  <sky-box-header>
+    <h2 class="sky-font-heading-2">
+      <span>Box header</span>
+      <sky-help-inline></sky-help-inline>
```

Comment:
I would expect the help inline to be a direct child of `sky-box-header`. This may mean we need to add styles to child elements of `sky-box-header` so they're `display: inline`. We can discuss in our next flex.

Reply:
There was an existing style to drop margins from child headings and I added `display: inline;` to that rule.